src/ImageTransformations.jl

```diff
@@ -1,7 +1,7 @@
 __precompile__()
 module ImageTransformations
 
-using CoordinateTransformations, Interpolations, OffsetArrays, StaticArrays, Colors, ColorVectorSpace, ImageCore
+using CoordinateTransformations, Interpolations, OffsetArrays, StaticArrays, FixedPointNumbers, Colors, ColorVectorSpace, ImageCore
```

Comment:
Tony will yell at us if we don't put `FixedPointNumbers` in `REQUIRE` if we use it directly